plugins/org.eclipse.fordiac.ide.typemanagement/src/org/eclipse/fordiac/ide/typemanagement/refactoring/rename/RenameTypeRefactoringParticipant.java

```diff
@@ -138,7 +138,7 @@ private CompositeChange createStructDataChange(final DataTypeEntry dataTypeEntry
 		final CompositeChange parentChange = new CompositeChange(
 				MessageFormat.format(Messages.Refactoring_RenameFromTo, typeEntry.getTypeName(), newName));
 		parentChange.add(new UpdateTypeEntryChange(file, typeEntry, newName, oldName));
-		final CompositeChange structUsageChanges = new CompositeChange("Refactoring struct users:");
+		final CompositeChange structUsageChanges = new CompositeChange("Refactoring struct users:"); //$NON-NLS-1$
```

Comment:
I think this is a string shown to the user. Please do not mark it as non translate able but export it.